docs/user-guide-sharing/validating-files.md

```diff
@@ -45,4 +45,11 @@ Passing all of these levels of validation can sometimes be tricky. If you have a
 
 ## Validating BIDS Files
 
-Once your files are converted to the BIDS standard, perform validation of the files using the [BIDS validator](https://bids.neuroimaging.io/tools/validator.html).
+Once your files are converted to the BIDS standard,
+
+  - ensure that you have `.bidsignore` file containing `dandiset.yaml`, since that DANDI-specific file is not part of BIDS specification;
```

Comment:
```suggestion
  - ensure that you have a `.bidsignore` file containing `dandiset.yaml`, since that DANDI-specific file is not part of the BIDS specification;
```